zipline-kotlin-plugin-hosted/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ plugins {
 dependencies {
   compileOnly(kotlin("compiler"))
 
-  kapt("com.google.auto.service:auto-service:1.0")
+  annotationProcessor("com.google.auto.service:auto-service:1.0")
```

Comment:
I have a hard time believing this still works correctly. What was the issue?

Reply:
Not sure, my remote IntelliJ was running on Ubuntu and couldn't build. Not sure if I saved the error anywhere unfortunately.